### DISCUSSION TITLE
Is there a Runge's phenomenon in SeisSol?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#869] [Q&A]
Although I use the same example simulation, using different orders of basis functions, when the fourth-order basis functions work normally, the sixth-order or seventh-order scheme encountered `NAN/INF` problem, but I can not find the reason, theoretically the fourth-order scheme can successfully work, grids should not be a problem, but the sixth-order basis functions or seventh-order basis functions encountered the problem, please ask you know whether there is  [Runge's phenomenon](https://en.wikipedia.org/wiki/Runge%27s_phenomenon)  ?

**Comment by Thomas-Ulrich** [comment]
larger order typically need lower CFL.
I would try CFL 0.45, 0.4 or even 0.25 and see if that works.

**Reply by buzhiqiuyuebai** [reply]
> larger order typically need lower CFL. I would try CFL 0.45, 0.4 or even 0.25 and see if that works.

Thakns, I will try to test it with lower CFL number.

**Comment by sebwolf-de** [comment]
We do not have the Runge phenomenom, as we use a modal basis. To add to @Thomas-Ulrich answer, here is a discussion about the CFL condition: https://github.com/SeisSol/SeisSol/issues/277.  For the order 6 or 7 schemes, CFL=0.5 is theoretically instable.

**Reply by krenzland** [reply]
> We do not have the Runge phenomenom

I mean, yes, but that's only true in the most technical sense. Of course we have numerical oscillations, as does every high-order code. However, they typically do not lead to inf/nans, as our PDE is linear and because we do not have any strict requirements of say positivity (e.g. the stress that we model can be both negative or positive, as we only model the perturbations).

For this issue here, the CFL condition is the most likely culprit.
